lib/DDG/Spice/Mtg.pm

```diff
@@ -0,0 +1,27 @@
+package DDG::Spice::Mtg;
+
+# mtg <card name> Returns infos on the Magic The Gathering Card
+
+use DDG::Spice;
+
+name "Mtg Card Search";
+description "mtg <card name> Returns infos on the Magic The Gathering Card";
```

Comment:
Last one! Please change it to just "Returns information about Magic The Gathering Cards"